kiliautoml/utils/constants.py

```diff
@@ -1,10 +1,12 @@
 import os
+from pathlib import Path
 from typing import Dict, List
 
 from typing_extensions import Literal, TypedDict
 
 HOME = os.path.join(os.getenv("HOME"), ".cache", "kili", "automl")  # type: ignore
 
+AUTOML_REPO_ROOT = Path(__file__).resolve().parents[2]
```

Comment:
I am not sure that this will work if automl is installed as a package and used from another directory.

Reply:
but anyway no big deal since we should get rid of Yolo as submodule moving forward